index.html

```diff
@@ -8,65 +8,69 @@
     <link rel="stylesheet" href="index.css">
 </head>
 <body>
-   <header>
-    <div class="logo"><p>John Doe</p></div>
-    <div class="menu"><img src="./images/Union.png" alt=""></div>
-   </header>
+  <main>
+    <header>
+      <div class="logo"><p>John Doe</p></div>
```

Comment:
- [ ] Kindly align your logo to the center of your toolbar to match the original design

![image](https://user-images.githubusercontent.com/98453979/194649452-d1fa6152-4524-4dd0-adac-1832459636ca.png)

### Checkout figma

![image](https://user-images.githubusercontent.com/98453979/194649519-f384d590-a877-4321-aff7-0159ff9bd8c2.png)

- [ ] Kindly resize the height of your image on your `multi-post stories section`  to match the Figma design. `hint` reduce the height to make it rectangular. Also, ensure that the spacing between the elements is uniform.

![image](https://user-images.githubusercontent.com/98453979/194650955-8809d3c0-02fe-422f-95ff-0d72e68820af.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/194651091-6711cde2-78b8-4bcd-b4e5-9d1ab1812efc.png)

- [ ] Kindly space elements on your `about section` uniformly to match the original design.

![image](https://user-images.githubusercontent.com/98453979/194651835-bdf6cc8b-5d00-4730-a50a-e050c1ae7190.png)

### Checkout figma

![image](https://user-images.githubusercontent.com/98453979/194651916-6d64beee-8c0b-4531-bf44-f82f327bc7ce.png)

- [ ] Kindly give your `get in touch` button on the contact section a `margin-top` to match the original design

![image](https://user-images.githubusercontent.com/98453979/194652234-6d7de79a-1806-48f0-b90b-501cdd78face.png)

- [ ] Kindly align the social icons on the `footer section` to the center to match the original design.

![image](https://user-images.githubusercontent.com/98453979/194652703-09af4acf-0492-48f6-9047-30511ae59edb.png)